joblib/test/test_memory.py

```diff
@@ -451,7 +451,7 @@ def test_call_and_shelve(tmpdir):
         assert result.get() == 5
 
         result.clear()
-        assert_raises(KeyError, result.get)
+        raises(KeyError, result.get)
```

Comment:
@lesteve for example, statements like this will seem weird. I did not use a `with` statement where regex is not checked. What do you think about this ?